lib/parameters.py

```diff
@@ -801,6 +801,25 @@ def gen_vp9_vbr_parameters(spec):
   params = gen_vp9_vbr_variants(spec)
   return keys, params
 
+def gen_vp9_seek_variants(spec):
+  for case, params in spec.items():
+    variants = params.get("variants", dict()).get("seek", None)
+
+    if variants is None:
+      keys = ["ipmode", "qp", "quality", "slices", "refmode", "looplvl", "loopshp", "seek"]
+      product = itertools.product([0, 1], [14, 28], [1, 4, 7], [1, 3], [0, 1], [0, 16, 63], [0, 4, 7])
+      variants = [dict(zip(keys, vals)) for vals in product]
```

Comment:
```suggestion
    variants = params.get("variants", dict()).get("seek", [])
```

...don't autogenerate cases, please.